src/dex-ui/pages/dao/MemberUpdate/MemberUpdate.tsx

```diff
@@ -0,0 +1,266 @@
+import { useLocation, useNavigate } from "react-router-dom";
+import { TransactionResponse } from "@hashgraph/sdk";
+import { Button, Flex, Spacer, Text } from "@chakra-ui/react";
+import { useSteps } from "chakra-ui-steps";
+import { useSendProposeTransaction } from "@hooks";
+import { FormProvider, useForm } from "react-hook-form";
+import {
+  MemberUpdateForm,
+  MemberOperationsType,
+  AddMemberForm,
+  DeleteMemberForm,
+  ReplaceMemberForm,
+  ChangeThresholdForm,
+} from "./types";
+import { Page } from "@dex-ui/layouts";
+import { Color, LoadingDialog, StepperV2 } from "@dex-ui-components";
+import { AddNewMemberDetailsForm } from "./AddNewMemberDetailsForm";
+import { AddNewMemberReviewForm } from "./AddNewMemberReviewForm";
+import { Paths } from "@dex-ui/routes";
+import { Member } from "../DAODetailsPage/MultiSigDAODashboard";
+import { WarningIcon } from "@chakra-ui/icons";
+import { DeleteMemberDetailsForm } from "./DeleteMemberDetailsForm";
+import { DeleteMemberReviewForm } from "./DeleteMemberReviewForm";
+import { ReplaceMemberDetailsForm } from "./ReplaceMemberDetailsForm";
+import { ReplaceMemberReviewForm } from "./ReplaceMemberReviewForm";
+import { ChangeThresholdDetailsForm } from "./ChangeThresholdDetailsForm";
+import { ChangeThresholdReviewForm } from "./ChangeThresholdReviewForm";
+
+interface MemberUpdateType {
+  operationType: string;
+}
+
+export function MemberUpdate(props: MemberUpdateType) {
```

Comment:
This logic for the Settings form wizards should be decoupled from each other. The components can be grouped under a folder for each of the following domains: Delete Member forms, Replace Member forms, Change Threshold forms, Add Member forms. Creating a dependency on a single component for all the forms creates unnecessary conditional logic and increases the complexity of refactoring, recomposing, and (in the future) writing tests for the components. 

Ideally, each of these forms will be nested under a dedicated <Route> component. The URL will update according to where the user is in the form flows. This allows the user to directly access any page in the wizard/forms via URL. Take a look at this `router.tsx` in this PR to see how this is accomplished: https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/pull/146/files. To incorporate these changes, the prior PR with the latest version of `react-router-dom` will need to be merged. For now, try to decouple this file as much as possible - we can refactor in a separate PR with the router approach.

Reply:
Yeah correct, I made since the UI for the forms are exactly same with just one field plus minus, may be once we merge the new route PR we can quickly discuss the approach on this.